f5-bigiq-generate-cert-with-SAN/README.md

```diff
@@ -0,0 +1,18 @@
+Installation instructions
+-------------------------
+
+Download the script on BIG-IQ CM.
+
+```
+curl https://raw.githubusercontent.com/f5devcentral/f5-big-iq-pm-team/master/f5-bigiq-generate-cert-with-SAN/generate-self-signed-cert > f5-generate-self-signed-cert.sh
+chmod +x f5-generate-self-signed-cert.sh
```

Comment:
As we execute the download and execute the script at the same time, we could remove the Download part (line 4-9)